### PR TITLE
UI - Payment Fail Banner and ApplicationDetails Unit Test

### PR DESCRIPTION
**Pull request by shaangill025**
*Issue:*

- https://github.com/bcgov/entity/issues/21996

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-283-oopj1wep.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-283-oopj1wep.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-283-oopj1wep.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-283-oopj1wep.web.app

File: strr-web/components/bcros/form-section/BusinessDetails.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div data-test-id="form-section-business-details">
-    <BcrosFormSection :title="t('common.formLabels.businessDetails')">
+    <BcrosFormSection :title="t('common.formLabels.businessDetails')" class="!ml-7">
```
Comment (dimak1) [inline strr-web/components/bcros/form-section/BusinessDetails.vue:3]: Is there another way to do it without enforcing the margin? I recently refactored and it was looking ok.

File: strr-web/tests/mocks/mockApplication.ts [hunk 2]
```diff
@@ -94,40 +94,44 @@ export const mockApplicationDetails: HostApplicationDetailsI = {
...
+    rentalUnitSpaceType: 'ENTIRE_HOME',
+    isUnitOnPrincipalResidenceProperty: true,
+    hostResidence: 'SAME_UNIT',
+    numberOfRoomsForRent: 1
   },
   registrationType: RegistrationTypeE.HOST,
   propertyManager: mockPropertyManager
 }
 
+const mockApplicationHeader = {
```
Comment (dimak1) [inline strr-web/tests/mocks/mockApplication.ts:107]: Mock needs a type.

**Review by dimak1 — COMMENTED** [review]
Some initial comments.

File: strr-web/components/bcros/form-section/BusinessDetails.vue (hunk 1 above)
Comment (shaangill025) [reply strr-web/components/bcros/form-section/BusinessDetails.vue]: After discussion, this has been removed.

File: strr-web/tests/mocks/mockApplication.ts (hunk 2 above)
Comment (shaangill025) [reply strr-web/tests/mocks/mockApplication.ts]: Fixed

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-283-oopj1wep.web.app

**Pull request merged**